pkgs/development/libraries/AntTweakBar/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, unzip, xlibs, mesa }:
```

Comment:
By the way, in general you can ask for `libX11` and `callPackage` will find it in `xlibs` for you. Top-level `callPackage` is configured to search top-level and `xlibs` (but nowhere else)